src/main/java/io/cdap/plugin/format/DBTableName.java

```diff
@@ -0,0 +1,27 @@
+package io.cdap.plugin.format;
+
+/**
+ * Information about a DB table name.
+ */
+public class DBTableName {
+  private String db;
+  private String tableName;
```

Comment:
nit: should be consistent and use 'dbName' and 'tableName', or 'db' and 'table'. I think it's better without the 'Name'

Reply:
done